source/Octopus.Tentacle/Communications/TentacleCommunicationsModule.cs

```diff
@@ -20,7 +20,16 @@ protected override void Load(ContainerBuilder builder)
             {
                 var configuration = c.Resolve<ITentacleConfiguration>();
                 var services = c.Resolve<IServiceFactory>();
-                var halibutRuntime = new HalibutRuntimeBuilder().WithServiceFactory(services).WithServerCertificate(configuration.TentacleCertificate).Build();
+                var halibutRuntime = new HalibutRuntimeBuilder()
+                    .WithServiceFactory(services)
+                    .WithServerCertificate(configuration.TentacleCertificate)
+                    .WithMessageSerializer(serializerBuilder => serializerBuilder.WithSerializerSettings(settings =>
```

Comment:
Register the new binders to performing the mapping to/from assembly and namespace.